scripts/importData/database.js

```diff
@@ -1,5 +1,8 @@
 const log = require('@kth/log')
 const nodeMongo = require('@kth/mongo')
+const dotenv = require('dotenv')
```

Comment:
I guess this was part of the confusion when I tried it as well as this wasn't part of the other PR. In the script Benni created he used the `--env` param to node to load a specific env-fil, and --env needs node 20. 

I like Benni approach with a separate .env-file and use of --env since it will be super clear which env-file that is used, and it will be less likely that you make a misstake to use wrong mongo db instance. 

What do you think? We can discuss this further tomorrow.

Reply:
Yes, now I think I remember that the reason for why I added this was because I got the connection string error as well when I tried to connect to my local db, but I think that Benni's approach sounds better! I'll be happy to discuss it tomorrow to understand it a bit better as well.